licenses.d2iq.yaml

```diff
@@ -45,6 +45,7 @@ ignore:
   - docker.io/bitnami/kubectl:1.22.0
   - docker.io/bitnami/kubectl:1.24.1
   - docker.io/bitnami/kubectl:1.26.4
+  - docker.io/bitnami/kubectl:1.28.5
```

Comment:
is this being bumped due to CVEs? 2.7 is on 1.27 so I wonder if we should use 1.27 kubectl images instead of 1.28 🤔

Reply:
Changed in 2303943c5a55ed173035c31b3e5e96999058f5f8